TLM/TLM/UI/IncompatibleModsPanel.cs

```diff
@@ -139,25 +139,16 @@ public void Initialize()
             blurEffect.size = new Vector2(resolution.x, resolution.y);
             blurEffect.absolutePosition = new Vector3(0, 0);
             blurEffect.SendToBack();
-            blurEffect.eventPositionChanged += OnBlurEffectPositionChange;
-            blurEffect.eventZOrderChanged += OnBlurEffectZOrderChange;
-            blurEffect.opacity = 0;
-            blurEffect.isVisible = true;
-            ValueAnimator.Animate("ModalEffect", delegate (float val) { blurEffect.opacity = val; }, new AnimatedFloat(0f, 1f, 0.7f, EasingType.CubicEaseOut));
+            if (blurEffect != null)
+            {
+                blurEffect.isVisible = true;
+                ValueAnimator.Animate("ModalEffect", delegate (float val) { blurEffect.opacity = val; }, new AnimatedFloat(0f, 1f, 0.7f, EasingType.CubicEaseOut));
+            }
 
             // Make sure modal dialog is in front of all other UI
             BringToFront();
         }
 
-        private void OnBlurEffectPositionChange(UIComponent component, Vector2 position) {
```

Comment:
why is that removed?

Reply:
Wrong version. Reapplied changes to this file only